include/exotic/cester.h

```diff
@@ -917,42 +935,46 @@ static __CESTER_INLINE__ void print_test_case_result(TestCase* test_case) {
         CESTER_DELEGATE_FPRINT_STR((CESTER_FOREGROUND_WHITE), "\n");
     } else {
-	if (test_case->execution_status == CESTER_RESULT_SUCCESS) {
-	    #ifdef _WIN32
-            CESTER_DELEGATE_FPRINT_STR((CESTER_FOREGROUND_GREEN), "+ ");
-	    #else
-            #ifdef __STDC_VERSION__
-                CESTER_DELEGATE_FPRINT_STR((CESTER_FOREGROUND_GREEN), "\u2713 ");
-            #else
+    	if (test_case->execution_status == CESTER_RESULT_SUCCESS) {
+	        #ifdef _WIN32
                 CESTER_DELEGATE_FPRINT_STR((CESTER_FOREGROUND_GREEN), "+ ");
-            #endif
-	    #endif
-	} else {
-	    #ifdef _WIN32
-            CESTER_DELEGATE_FPRINT_STR((CESTER_FOREGROUND_RED), "- ");
-	    #else
-            #ifdef __STDC_VERSION__
-                CESTER_DELEGATE_FPRINT_STR((CESTER_FOREGROUND_RED), "X ");
-            #else
+	        #else
+                #ifdef __STDC_VERSION__
+                    CESTER_DELEGATE_FPRINT_STR((CESTER_FOREGROUND_GREEN), "\u2713 ");
+                #else
+                    CESTER_DELEGATE_FPRINT_STR((CESTER_FOREGROUND_GREEN), "+ ");
+                #endif
+	        #endif
+	    } else {
+	        #ifdef _WIN32
                 CESTER_DELEGATE_FPRINT_STR((CESTER_FOREGROUND_RED), "- ");
-            #endif
-	    #endif
-	    print_color = __CESTER_CAST_CHAR_ARRAY__ CESTER_FOREGROUND_RED;
-	}
-	CESTER_DELEGATE_FPRINT_STR((print_color), "(");
-	CESTER_DELEGATE_FPRINT_DOUBLE_2((print_color), (test_case->execution_time > 60 ? (test_case->execution_time / 60) : test_case->execution_time));
-	CESTER_DELEGATE_FPRINT_STR((print_color), (test_case->execution_time > 60 ? "m" : "s"));
-        CESTER_DELEGATE_FPRINT_STR((print_color), ") ");
-        CESTER_DELEGATE_FPRINT_STR((print_color), (superTestInstance.format_test_name == 1 ? cester_str_replace(test_case->name, '_', ' ') : test_case->name ));
-        CESTER_DELEGATE_FPRINT_STR((print_color), "\n");
+	        #else
+                #ifdef __STDC_VERSION__
+                    CESTER_DELEGATE_FPRINT_STR((CESTER_FOREGROUND_RED), "X ");
+                #else
+                    CESTER_DELEGATE_FPRINT_STR((CESTER_FOREGROUND_RED), "- ");
+                #endif
+	        #endif
+	        print_color = __CESTER_CAST_CHAR_ARRAY__ CESTER_FOREGROUND_RED;
+	    }
+        #ifndef CESTER_NO_TIME
+	        CESTER_DELEGATE_FPRINT_STR((print_color), "(");
+	        CESTER_DELEGATE_FPRINT_DOUBLE_2((print_color), (test_case->execution_time > 60 ? (test_case->execution_time / 60) : test_case->execution_time));
+	        CESTER_DELEGATE_FPRINT_STR((print_color), (test_case->execution_time > 60 ? "m" : "s"));
+            CESTER_DELEGATE_FPRINT_STR((print_color), ") ");
+            CESTER_DELEGATE_FPRINT_STR((print_color), (superTestInstance.format_test_name == 1 ? cester_str_replace(test_case->name, '_', ' ') : test_case->name ));
+            CESTER_DELEGATE_FPRINT_STR((print_color), "\n");
+        #endif

```

Comment:
This check completely prevents the test case name from being displayed in the text output, only the checkmark is displayed. Adjust it to suppress the time only.

Moving line 975 and 976 out of #ifndef CESTER_NO_TIME should fix this

Except there is issue printing out the test case name on the platform

Reply:
Ah! That's where my problem was :D Thanks. I'll fix it.

Reply:
Done. Sorry for the delay.

Reply:
Looks like it works now:

![image](https://user-images.githubusercontent.com/7281574/84113098-d46d7b00-a9de-11ea-8b13-0e3710558232.png)